pkg/xds/envoy/listeners/v3/pipe_listener_configurer.go

```diff
@@ -0,0 +1,22 @@
+package v3
+
+import (
+	envoy_core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	envoy_api "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
+)
+
+type PipeListenerConfigurer struct {
```

Comment:
unix domain sockets and pipes are a bit different, not sure why envoy calls them the same 🤷  https://www.baeldung.com/linux/kernel-counterparts-explained#comparison-and-distinctions but we can stick to [Envoy naming here](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-pipe).